docker-images/openemr/run_openemr.sh

```diff
@@ -1,9 +1,53 @@
 #!/bin/sh
+# to be run by root with OpenEMR root dir as the CWD
 
 # allows customization of openemr credentials, preventing the need for manual setup
 #  - Setting db parameters via MYSQL_HOST, MYSQL_USER, MYSQL_PASS
 #  - Setting first-time account creation via OE_USER, OE_PASS
 #  - All of these must be passed to automate the first-time setup process
+set -e
+
+auto_setup() {
+    chmod -R 600 .
+    php auto_configure.php -f \
+        iuser=$OE_USER \
+        iuserpass=$OE_PASS \
+        login=$MYSQL_USER \
+        rootpass=$MYSQL_ROOT_PASS \
+        pass=$MYSQL_PASS \
+        server=$MYSQL_HOST \
+
+    #do I need to activate mod_rewrite?
+    echo "OpenEMR configured. Setting user 'www' as owner of openemr/ and setting file/dir permissions to 400/500"
+    #set all directories to 500
+    find . -type d -print0 | xargs -0 chmod 500
+    #set all file access to 400
+    find . -type f -print0 | xargs -0 chmod 400
+
+    echo "Default file permissions and ownership set, allowing writing to specific directories"
+    chmod 700 run_openemr.sh
+    # Set file and directory permissions
+    #chmod 666 sites/default/sqlconf.php why?
```

Comment:
After the install, agree this is treated just like all the other normal scripts(just needed to be writable during the install as you did above).

Reply:
I'll go ahead and remove that comment then!